StRoot/StBFChain/BigFullChain.h

```diff
@@ -1377,12 +1378,38 @@ Bfc_st BFC[] = { // standard chains
   {"------------","-----------","-----------","------------------------------------------","","","",kFALSE},
   {"Generators  ","-----------","-----------","------------------------------------------","","","",kFALSE},
   {"------------","-----------","-----------","------------------------------------------","","","",kFALSE},
-  {"stargen",     "","", "gen_T,sim_T"/*+++*/,                     "", "libVMC.so,libStarGeneratorUtil.so,libStarGeneratorEvent.so,libStarGeneratorBase.so,libStarGeneratorFilt.so,libMathMore.so","STAR Generator BASE",false},
+  {"stargen.filt", "","", "gen_T,sim_T"/*+++*/,                     "", "libVMC.so,libStarGeneratorFilt.so,libMathMore.so","STAR Generator BASE",false},
+  {"stargen.util", "","", "gen_T,sim_T"/*+++*/,                     "", "libVMC.so,libStarGeneratorUtil.so,libMathMore.so","STAR Generator BASE",false},
+  {"stargen.event","","", "gen_T,sim_T"/*+++*/,                     "", "libVMC.so,libStarGeneratorEvent.so,libMathMore.so","STAR Generator BASE",false},
+  {"stargen.base", "","", "gen_T,sim_T"/*+++*/,                     "", "libVMC.so,libStarGeneratorBase.so,libMathMore.so","STAR Generator BASE",false},
+  {"stargen",     "","",  "stargen.util,stargen.event,stargen.base"/*+++*/,                     "", "","STAR Generator BASE",false},
```

Comment:
Note: stargen loads have an unmet dependency in St_geant_Maker.  Future commits will resolve this.